Makefile

```diff
@@ -0,0 +1,8 @@
+.PHONY: dependencies static_analysis
+
+dependencies:
+	pip install --upgrade pip
```

Comment:
Shouldn't that be pip3?
As we are targeting only python3

Reply:
Pushed a fix